mautic/mautic
# Prevented fatal errors when core fields are disabled

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | Y
| New feature?  | N 
| BC breaks?    | N
| Deprecations? | N
| Fixed issues  |  

## Description
This supercedes #997. When a core field is disabled, the contact form throws fatal errors and the list views throw notices.

## Steps to reproduce the bug (if applicable)

Disable a core field (name, title, address1, etc). Try to open the contact form and you should be greeted with a fatal error. 

## Steps to test this PR

Apply the PR and repeat the above. This time the contact should be editable. Email is prevented from being unpublished since it is key to all of Mautic.
